src/TaskRunner/Commands/ToolCommands.php

```diff
@@ -183,13 +182,11 @@ public function componentCheck(array $options = [
                 $msg = 'Failed the components check, please verify the report and update the project.';
                 $msg .= "\nSee the list of packages at https://webgate.ec.europa.eu/fpfis/qa/package-reviews.";
                 $this->io()->error($msg);
-                $status = 1;
+                return 1;
```

Comment:
Remove the return, with this return it will not print the notice. Previous code looks ok